store/storetest/channel_store.go

```diff
@@ -1153,36 +1152,31 @@ func testChannelStoreGetAllChannels(t *testing.T, ss store.Store, s SqlSupplier)
 	_, err = ss.Channel().Save(&c5, -1)
 	require.Nil(t, err)
 
-	cresult := <-ss.Channel().GetAllChannels(0, 10, store.ChannelSearchOpts{})
-	list := cresult.Data.(*model.ChannelListWithTeamData)
+	list, _ := ss.Channel().GetAllChannels(0, 10, store.ChannelSearchOpts{})
```

Comment:
nit: would be nice to add here the `require.Nil(t, err)` for this call (and for the rest of this calls in the file).

Reply:
Ok, I added this suggestion